bedrock/firefox/tests/test_base.py

```diff
@@ -286,7 +286,7 @@ def test_fx_beta_68_0_trailhead_non_locales(self, render_mock):
 
     @override_settings(DEV=True)
     def test_fx_dev_browser_35_0_a2_whatsnew(self, render_mock):
-        """Should defaultr whatsnew template"""
+        """Should default whatsnew template"""
```

Comment:
I'm not sure what this comment is supposed to communicate. Can you update it for clarity please?

Reply:
Doh sorry, will update!